clubs/models.py

```diff
@@ -71,6 +61,104 @@ class Club(models.Model):
         null=True
     )
 
+    occupation = models.TextField(
+        help_text=("Club captain's occupation."),
+        blank=False,
+        null=True
+    )
+
+    regional_coordinate = models.TextField(
+        help_text=("Regional Coordinate associated with club captain."),
+        blank=True,
+        null=True
+    )
+
+    hosting_reason = models.TextField(
+        help_text=("Reason to host a Mozilla club."),
+        blank=False,
+        null=True
+    )
+
+    how_they_heard = models.TextField(
+        help_text="How did the applicant hear about Current approval status of the club.",
+        blank=False,
+        null=True
+    )
+
+
+    '''
+        "Page 2" club-specific values
+    '''
+
+    name = models.CharField(
+        help_text="The name of the club.",
+        max_length=100
+    )
+
+    description = models.TextField(
+        help_text="Description of the club.",
+        blank=False,
+        null=True
+    )
+
+    venue = models.TextField(
+        help_text="Where does this club meet.",
+        blank=False,
+        null=True
+    )
+
+    # this is a range, and possibly "other" as user-supplied
+    # information. As such, this is a text field.
+    frequency = models.TextField(
+        help_text=("How frequently this club meets."),
+        blank=False,
+        null=True
+    )
+
+    age_range = models.TextField(
+        help_text=("The club member age range."),
+        blank=False,
+        null=True
+    )
+
+    club_size = models.TextField(
+        help_text=("The number of club members."),
+        blank=False,
+        null=True
+    )
+
+    member_occupation = models.TextField(
+        help_text=("Club member occupations."),
+        blank=False,
+        null=True
+    )
+
+    club_topics = models.TextField(
+        help_text=("Club topics/subjects."),
+        blank=False,
+        null=True
+    )
+
+    '''
+      Optional application values
+    '''
+
+    affiliation = models.TextField(
+        help_text="The affiliated institution or organization.",
+        blank=True,
+        null=True
+    )
+
+    website = models.URLField(
```

Comment:
oh, what? Yeah that's a little low, good thing you spotted that. The browser-legal value for that would technically be 4096, wondering what a good value is for us to use here.